tools/Makefile

```diff
@@ -77,20 +77,19 @@ ifneq ($(strip $(CUPY_VERSION)),)
 endif
 
 warp-ctc.done: espnet.done
-	rm -rf warp-ctc
-	git clone https://github.com/espnet/warp-ctc.git
-	# Note(kamo): Requires gcc>=4.9 to build extensions with pytorch>=1.0
-	if . venv/bin/activate && python -c 'import torch as t;assert t.__version__[0] == "1"' &> /dev/null; then \
-        . venv/bin/activate && python -c "from distutils.version import LooseVersion as V;assert V('$(GCC_VERSION)') >= V('4.9'), 'Requires gcc>=4.9'"; \
-	fi
-	if . venv/bin/activate && python -c 'import torch as t;assert t.__version__[0] == "1"' &> /dev/null; then \
-        cd warp-ctc; git checkout -b pytorch-1.0.0 remotes/origin/pytorch-1.0.0; \
-	elif . venv/bin/activate && python -c 'import torch as t;assert t.__version__[0] == "0"' &> /dev/null; then \
-        cd warp-ctc; git checkout -b pytorch-0.4.0 remotes/origin/pytorch-0.4; \
+	if . venv/bin/activate && python -c 'import torch as t;major=t.__version__.split(".")[0];assert major == "1"' &> /dev/null; then \
+		if [ ! -z "$(strip $(CUPY_VERSION))" ]; then \
```

Comment:
```suggestion
		if [ ! -z "$(strip $(CUDA_VERSION))" ]; then \
```
???

Reply:
https://github.com/espnet/espnet/blob/master/README.md#step-2-c-installation-for-cpu-only
This section says CUPY version should be cleared in CPU-only environment.